stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -16,5 +22,76 @@ protected String onGetNodeName(View element) {
         StringUtil.removePrefix(className, "android.view.",
         StringUtil.removePrefix(className, "android.widget."));
   }
+
+  @Override
+  protected void onCopyAttributes(View element, AttributeAccumulator attributes) {
+    String id = getIdAttribute(element);
+    if (id != null) {
+      attributes.add(ID_ATTRIBUTE_NAME, id);
+    }
+  }
+
+  // Copied from Resources.resourceHasPackage because it's marked as @hide
+  private static boolean resourceHasPackage(int resid) {
```

Comment:
Tried skipping the check and running on a few different apps ... seems to work, why not?